lib/test/createIdentity.js

```diff
@@ -0,0 +1,147 @@
+const {
+  Transaction,
+  PrivateKey,
+  PublicKey,
+} = require('@dashevo/dashcore-lib');
+
+const DashPlatformProtocol = require('@dashevo/dpp');
+
+const stateTransitionTypes = require('@dashevo/dpp/lib/stateTransition/stateTransitionTypes');
+const IdentityCreateTransition = require('@dashevo/dpp/lib/identity/stateTransitions/identityCreateTransition/IdentityCreateTransition');
+const IdentityPublicKey = require('@dashevo/dpp/lib/identity/IdentityPublicKey');
+
+const wait = require('../wait');
+
+/**
+ *
+ * @param {DAPIClient} dapiClient
+ * @param {Address} faucetAddress
+ * @param {PrivateKey} faucetPrivateKey
+ * @param {Address} address
+ * @return {Promise<string>}
+ */
+async function fundAddress(dapiClient, faucetAddress, faucetPrivateKey, address) {
+  const { items: inputs } = await dapiClient.getUTXO(faucetAddress.toString());
```

Comment:
How is this going to work once dashevo/dapi/pull/218 removes `getUTXO`?

Reply:
We brought it back.